nessie/src/test/java/org/apache/iceberg/nessie/TestMultipleClients.java

```diff
@@ -98,12 +98,12 @@ public void testLoadNamespaceMetadata() {
 
   @Test
   public void testListTables() {
-    catalog.createTable(TableIdentifier.parse("foo.tbl1"), schema);
+    createTable(catalog, TableIdentifier.parse("foo.tbl1"), schema);
```

Comment:
why not just call `catalog.createNamespace(Namespace.of("foo"), Collections.emptyMap());` at the beginning of these methods?